src/discordmessageprocessor.ts

```diff
@@ -79,10 +79,27 @@ export class DiscordMessageProcessor {
     public async FormatEdit(
         oldMsg: Discord.Message,
         newMsg: Discord.Message,
+        link?: string,
     ): Promise<DiscordMessageProcessorResult> {
-        // TODO: Produce a nice, colored diff between the old and new message content
+        // https://matrix.to/#/!room/$event
+        oldMsg.embeds = []; // we don't want embeds on old msg
+        const oldMsgParsed = await this.FormatMessage(oldMsg);
+        const newMsgParsed = await this.FormatMessage(newMsg);
+        const result = new DiscordMessageProcessorResult();
+        result.body = `*edit:* ~~${oldMsgParsed.body}~~ -> ${newMsgParsed.body}`;
+        result.msgtype = newMsgParsed.msgtype;
         oldMsg.content = `*edit:* ~~${oldMsg.content}~~ -> ${newMsg.content}`;
-        return this.FormatMessage(oldMsg);
+        const linkStart = link ? `<a href="${escapeHtml(link)}">` : "";
+        const linkEnd = link ? "</a>" : "";
+        const MAX_MSG_LENGTH = 50;
```

Comment:
needs to be more descriptive like "MAX_EDIT_MSG_LENGTH" and declared at the top of the file.